master/buildbot/process/buildstep.py

```diff
@@ -566,6 +560,14 @@ def setRenderable(res, attr):
             # we describe ourselves only when renderables are interpolated
             self.realUpdateSummary()
 
+            # check doStepIf (after rendering)
```

Comment:
Hi @trefzer 
Here is the documentation on how to run/write unit tests for buildbot.

http://docs.buildbot.net/latest/developer/tests.html
You can start by modifying this test
https://github.com/buildbot/buildbot/blob/master/master/buildbot/test/unit/test_process_buildstep.py#L242
and run ```trial buildbot.test.unit.test_process_buildstep.TestBuildStep.test_doStepIf_false```

Please remember that this project is entirely run by volunteers. Unit tests are an essential way to keep our code stable for the benefit of the whole community. We cannot accept contributions without unit test and docs update.

This code is fundamental and run by everybody. It is fundamental it is carefully tested.

Thank you for your understanding